src/main/scala/net/degoes/03-nulls.scala

```diff
@@ -27,19 +27,24 @@ object Nulls extends ZIOSpecDefault {
         test("apply") {
           import java.io.File
 
-          def parentOf(file: String) = new File(file).getParent
+          def parentOf(file: String): Option[File] = new File(file).getParent match {
```

Comment:
You can simplify it further because `Option(null)` returns `None`, so you don't need the pattern matching